README.md

```diff
@@ -1,6 +1,6 @@
-# DES Exposure Checker
+# Exposure Checker
 
-Crowdsourced image quality control for the Dark Energy Survey (DES). The code available here is for a web-app that asks users to identify flaws in survey images. The app loads and displays FITS image directly in the browser and works on desktop and mobile devices.
+Crowdsourced image quality control for DC2. The code available here is for a web-app that asks users to identify flaws in survey images. The app loads and displays FITS image directly in the browser and works on desktop and mobile devices.
```

Comment:
What do you think about being more general?

Reply:
OK !